src/main/groovy/net/foragerr/jmeter/gradle/plugins/worker/JMeterRunner.groovy

```diff
@@ -48,12 +48,6 @@ class JMeterRunner {
         argumentsList.toArray(new String[argumentsList.size()])
     }
 
-    /**
```

Comment:
Hi @foragerr The comment deletion was not on purpose, it was just something what happened when I did the fork. I had reverted it and it must appear again now.